JSTests/stress/regress-153522603.js

```diff
@@ -0,0 +1,6 @@
+//@ runDefault("--useFTLJIT=0", "--jitPolicyScale=0.1", "--watchdog=5000", "--watchdog-exception-ok")
```

Comment:
Is there a reason to use a `5s` watchdog? Seems like we could get away with a much smaller number like 200ms or something.

Reply:
Good call. I can repro locally easily enough in debug builds with 200ms. Note that I can't easily repro neither this test case or the original fuzz test in non-ASan release.